<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>bdMultiBlastWin</class>
 <widget class="QMainWindow" name="bdMultiBlastWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>466</width>
    <height>708</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>154</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>152</height>
       </size>
      </property>
      <property name="title">
       <string>Paths</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="2" column="2">
          <widget class="QPushButton" name="blastsPathBtn">
           <property name="styleSheet">
            <string notr="true">    QPushButton{
                                    color: white;
                                    border-radius: 4px;
                                    min-height: 30px;
                                    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #103050, stop: 1 #0d5ca6);
                                   }
    QPushButton:hover {
                                    color: white;
                                    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #13385d, stop: 1 #2570bb);
                                   }                                     
    QPushButton:pressed {
                                    color: white;
                                    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #0d5ca6, stop: 1 #2198c0);
                                   }  
    QPushButton:disabled{
                                    color: gray;
                                    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #616161, stop: 1 #292929);
                                   }</string>
           </property>
           <property name="text">
            <string>Browse</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label">
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Sunken</enum>
           </property>
           <property name="text">
            <string>Animations</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_2">
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Sunken</enum>
           </property>
           <property name="text">
            <string>Output</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QPushButton" name="mayaFilesPathBtn">
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>30</height>
            </size>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="styleSheet">
            <string notr="true">    QPushButton{
                                    color: white;
                                    border-radius: 4px;
                                    min-height: 30px;
                                    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #103050, stop: 1 #0d5ca6);
                                   }
    QPushButton:hover {
                                    color: white;
                                    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #13385d, stop: 1 #2570bb);
                                   }                                     
    QPushButton:pressed {
                                    color: white;
                                    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #0d5ca6, stop: 1 #2198c0);
                                   }  
    QPushButton:disabled{
                                    color: gray;
                                    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #616161, stop: 1 #292929);
                                   }</string>
           </property>
           <property name="text">
            <string>Browse</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="blastPath"/>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="animPath"/>
         </item>
         <item row="4" column="1">
          <widget class="QLineEdit" name="camPath"/>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_5">
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Sunken</enum>
           </property>
           <property name="text">
            <string>Camera</string>
           </property>
          </widget>
         </item>
         <item row="4" column="2">
          <widget class="QPushButton" name="camPathBtn">
           <property name="styleSheet">
            <string notr="true">    QPushButton{
                                    color: white;
                                    border-radius: 4px;
                                    min-height: 30px;
                                    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #103050, stop: 1 #0d5ca6);
                                   }
    QPushButton:hover {
                                    color: white;
                                    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #13385d, stop: 1 #2570bb);
                                   }                                     
    QPushButton:pressed {
                                    color: white;
                                    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #0d5ca6, stop: 1 #2198c0);
                                   }  
    QPushButton:disabled{
                                    color: gray;
                                    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #616161, stop: 1 #292929);
                                   }</string>
           </property>
           <property name="text">
            <string>Browse</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_6">
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Sunken</enum>
           </property>
           <property name="text">
            <string>Character</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="rigPath"/>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="rigPathBtn">
           <property name="styleSheet">
            <string notr="true">    QPushButton{
                                    color: white;
                                    border-radius: 4px;
                                    min-height: 30px;
                                    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #103050, stop: 1 #0d5ca6);
                                   }
    QPushButton:hover {
                                    color: white;
                                    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #13385d, stop: 1 #2570bb);
                                   }                                     
    QPushButton:pressed {
                                    color: white;
                                    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #0d5ca6, stop: 1 #2198c0);
                                   }  
    QPushButton:disabled{
                                    color: gray;
                                    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #616161, stop: 1 #292929);
                                   }</string>
           </property>
           <property name="text">
            <string>Browse</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>350</height>
       </size>
      </property>
      <property name="title">
       <string>Animations</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QTableWidget" name="mayaFilesTableWidget">
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <column>
          <property name="text">
           <string>Playblast</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>File name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Version</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="lastVersion">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Last Versions - only if the file name ends in 'v##'  ( experimental)</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="playblastBtn">
         <property name="styleSheet">
          <string notr="true">    QPushButton{
                                    color: white;
                                    border-radius: 4px;
                                    min-height: 30px;
                                    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #103050, stop: 1 #0d5ca6);
                                   }
    QPushButton:hover {
                                    color: white;
                                    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #13385d, stop: 1 #2570bb);
                                   }                                     
    QPushButton:pressed {
                                    color: white;
                                    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #0d5ca6, stop: 1 #2198c0);
                                   }  
    QPushButton:disabled{
                                    color: gray;
                                    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #616161, stop: 1 #292929);
                                   }</string>
         </property>
         <property name="text">
          <string>Batch Playblast Files</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="playblastOptionGrp">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>100</height>
       </size>
      </property>
      <property name="title">
       <string>Options</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="label_3">
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Sunken</enum>
           </property>
           <property name="text">
            <string>Format</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="formatComboBox">
           <property name="frame">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_4">
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Sunken</enum>
           </property>
           <property name="text">
            <string>Encoding</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="encodingComboBox"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QProgressBar" name="progressBar">
      <property name="value">
       <number>0</number>
      </property>
      <property name="textVisible">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>0</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionCopy_To_Preview">
   <property name="text">
    <string>Copy To Preview</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>rigPath</tabstop>
  <tabstop>animPath</tabstop>
  <tabstop>blastPath</tabstop>
  <tabstop>camPath</tabstop>
  <tabstop>rigPathBtn</tabstop>
  <tabstop>blastsPathBtn</tabstop>
  <tabstop>mayaFilesPathBtn</tabstop>
  <tabstop>camPathBtn</tabstop>
  <tabstop>playblastBtn</tabstop>
  <tabstop>formatComboBox</tabstop>
  <tabstop>encodingComboBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
